cl/api/models.py

```diff
@@ -44,28 +45,83 @@ class Webhook(AbstractDateTimeModel):
         default=0,
     )
 
+    class Meta:
+        indexes = [
+            models.Index(fields=["enabled"]),
+        ]
+
```

Comment:
Yeah, that's right. I did some tests adding some thousand records and in fact, this index didn't help more to the performance than indexes in `WebhookEvent` did it.

Reply:
It sounds like you added thousands of rows and then compared performance, but in case you don't know, if you ever want to see if an index is used, you can do an ANALYZE of the query and it'll tell you the query plan.